interfaces/cython/cantera/kinetics.pyx

```diff
@@ -176,9 +176,21 @@ cdef class Kinetics(_SolutionBase):
         self.kinetics.addReaction(rxn._reaction)
 
     def is_reversible(self, int i_reaction):
-        """True if reaction ``i_reaction`` is reversible."""
-        self._check_reaction_index(i_reaction)
-        return self.kinetics.isReversible(i_reaction)
+        """
+        True if reaction ``i_reaction`` is reversible.
+
+        .. deprecated:: 2.6
+
+            Replaced by property ``Reaction.reversible``.
+            Example: ``gas.is_reversible(0)`` is replaced by
+            ``gas.reaction(0).reversible``
+        """
+        rxn = self.reaction(i_reaction)
+        warnings.warn(
+            "Class method 'is_reversible' is deprecated and will be removed "
```

Comment:
Could you clarify? It is my understanding that classes have methods and properties?

Reply:
A class method is applied to the class before construction. This is technically an instance method. But for users in this context I don't think the distinction between a method and a property is important and probably just confusing, so I think it can be removed altogether.

Reply:
OK, so it was the 'class' that is the issue here. I'd prefer to leave 'method' and 'property' as we've turned methods that don't take arguments into properties. I'll just remove the `class' then ...

Reply:
... alright. Starting with the method name as requested :wink: